src/lib/mdns/minimal/responders/tests/TestPtrResponder.cpp

```diff
@@ -31,7 +31,9 @@ using namespace chip::Encoding;
 using namespace chip::Inet;
 using namespace mdns::Minimal;
 
+#if INET_CONFIG_ENABLE_IPV4
 constexpr uint16_t kMdnsPort = 5353;
```

Comment:
This doesn't seem right.  How is this conditional on IPv4?

Reply:
Test is only performed on IPv4 srv addresses, so compiler was complaining about constant never being used.

Actual good change would be to use IPv6 here as well, will see what can be done there.